arm-visualstudio/2014-04-01-preview/swagger/Csm.json

```diff
@@ -0,0 +1,867 @@
+      "get": {
+        "tags": [
+          "accountResourceOperations"
+        ],
+        "summary": "GetOperationResources",
+        "description": "",
+        "operationId": "getOperationResources_List",
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation succeeded. The response contains the possible operations on the Microsoft.VisualStudio resource provider.",
+            "schema": {
+              "$ref": "#/definitions/ResourceOperationsDescriptorWrapper"
+            }
+         }
+        },
+        "x-ms-examples": {
+            "getOperationResources_List": {
+                "$ref": "../examples/GetOperations.json"
+            }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/{resourceProviderNamespace}/{resourceType}": {
+      "get": {
+        "tags": [
+          "accountResourceGroup"
+        ],
+        "summary": "GetResources",
+        "description": "Gets the Visual Studio Team Services account resources linked to the specified Azure subscription.",
+        "operationId": "getResources_List",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "resourceGroupName",
+            "description": "Name of the resource group.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "resourceProviderNamespace",
+            "description": "The name of the ARM resource provider. This should currently always be set to 'Microsoft.VisualStudio'.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "resourceType",
+            "description": "Type of the resource. This should currently always be set to 'Account'.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation succeeded. The response contains the details of the Visual Studio Team Services account resources linked to the Azure subscription.",
+            "schema": {
+              "$ref": "#/definitions/ResourceGroupGetResponse"
+            }
+         }
+        },
+        "x-ms-examples": {
+            "getResources_List": {
+                "$ref": "../examples/GetResources_List.json"
+            }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/{resourceProviderNamespace}/{resourceType}/{resourceName}": {
+      "get": {
+        "tags": [
+          "accountResource"
+        ],
+        "summary": "GetResource",
+        "description": "Gets the Visual Studio Team Services account resource details.",
+        "operationId": "getResource",
```

Comment:
I've renamed all of the methods to include group names. Since making that change, I'm seeing this warning in VSCode:

`The child tracked resource, 'extension' with immediate parent 'AccountResourceGetResponse', must have a list by immediate parent operation.'`

Not quite sure what the issue is here since the immediate parent (Account) does have a List operation. Is something off with my naming?